cron/centAcl.php

```diff
@@ -112,15 +112,15 @@ function programExit($msg)
 
     if ($is_running == 0) {
         $DBRESULT = $pearDB->query(
-            "UPDATE cron_operation SET running = '1', time_launch = '" . time() .
+            "UPDATE cron_operation SET `running` = '1', `time_launch` = '" . time() .
             "' WHERE id = '$appID'"
```

Comment:
Is `$appID` a string from Centreon env or a variable from PHP ? If it is a PHP variable, I think it is more "visual" to separate from the query string, concatenating the variable. (correct form :smile: )

Reply:
Some of "system", "running" and "activate" field's names seems to be key words in mysql.
So I added back quote on them...